docs/node-operators/cli.md

```diff
@@ -194,44 +485,231 @@ bye
+```
+
+``` tab="cargo"
+cargo run -- node send --address=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=1234 --fee=1
 {"jsonrpc":"2.0","result":"a6c26804cf76e08ab379ea63e4aa046095dade2ae52fb3ecac90817583e61349","id":"1"}
 ```
 
 On error, returns the error message:
 
+``` tab="docker"
+docker exec -i witnet_node ./witnet node send --address=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=9999999999999999999 --fee=1
+{"jsonrpc":"2.0","error":{"code":-32603,"message":"Cannot build a transaction transferring more value than the current available balance: 9999999999999999999 + 1 > 39649999949502"},"id":"1"}
 ```
-witnet node send --pkh=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=9999999999999999999 --fee=1
+
+``` tab="binary"
+witnet node send --address=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=9999999999999999999 --fee=1
+{"jsonrpc":"2.0","error":{"code":-32603,"message":"Cannot build a transaction transferring more value than the current available balance: 9999999999999999999 + 1 > 39649999949502"},"id":"1"}
+```
+
+``` tab="cargo"
+cargo run -- node send --address=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=9999999999999999999 --fee=1
 {"jsonrpc":"2.0","error":{"code":-32603,"message":"Cannot build a transaction transferring more value than the current available balance: 9999999999999999999 + 1 > 39649999949502"},"id":"1"}
 ```
 
 Example with time lock set to 2019-10-01
 
+``` tab="docker"
+docker exec -i witnet_node ./witnet node send --address=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=1234 --fee=1 --time-lock=1569888000
+{"jsonrpc":"2.0","result":"e5b55ec4930f32383e63de9316238f369ee26d89d4375521071a885fc46b4c17","id":"1"}
+```
+
+``` tab="binary"
+witnet node send --address=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=1234 --fee=1 --time-lock=1569888000
+{"jsonrpc":"2.0","result":"e5b55ec4930f32383e63de9316238f369ee26d89d4375521071a885fc46b4c17","id":"1"}
 ```
-witnet node send --pkh=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=1234 --fee=1 --time-lock=1569888000
+
+``` tab="cargo"
+cargo run -- node send --address=twit1ulyzvnknjnndkfva636erkkp83wxhhwdfhptsr --value=1234 --fee=1 --time-lock=1569888000
 {"jsonrpc":"2.0","result":"e5b55ec4930f32383e63de9316238f369ee26d89d4375521071a885fc46b4c17","id":"1"}
 ```
 
+### sendRequest
+
+Send a serialized data request. An example of how to create such a request can be found in the [tutorial](https://docs.witnet.io/tutorials/bitcoin-price-feed/introduction/).
+
+```sh tab="docker"
+docker exec -i witnet_node ./witnet node sendRequest --hex "data request output serialized in hexadecimal format"
+```
+
+```sh tab="cargo"
+cargo run -- node sendRequest --hex "data request output serialized in hexadecimal format"
+```
+
+```sh tab="binary"
+witnet node sendRequest --hex "data request output serialized in hexadecimal format"
+```
+
+### server
+
+Run a Witnet node server.
+
+```sh tab="docker"
+docker exec -i witnet_node ./witnet node server
+```
```

Comment:
```suggestion
```text tab="docker"
It is usually not a good idea to run a second Witnet node server inside an existing Docker container. Please use a second Docker container instead.
```
```